src/Compatibility/Maps/src/Android/Compatibility.Maps.Android.csproj

```diff
@@ -17,6 +17,7 @@
     <ProjectReference Include="..\..\..\Core\src\Android\Compatibility.Android.csproj" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.Maui.Graphics" Version="6.0.100-preview.3.34" />
```

Comment:
I still think the version from Microsoft.Maui.Graphics needs to be in a single place. It looks like it is in 6 places?

Reply:
🍰